pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -504,9 +508,11 @@ ul {
   }
 }
 
-// Ordered lists
-// Step-by-step section
-ol.steps-list {
+// Step-by-step lists
+// These styles can be used with an ordered list as seen on code.org/professional-learning/middle-high
+// or an unordered list that uses icons as seen on code.org/curriculum/game-design.
```

Comment:
Added these new styles to avoid refactoring this section since it was intended to be an ordered list with number steps, but now it can work with icons too 🙌